packages/relay/src/lib/services/hapiService/hapiService.ts

```diff
@@ -101,7 +101,7 @@ export default class HAPIService {
     const currentDateNow = Date.now();
     this.initialTransactionCount = parseInt(process.env.HAPI_CLIENT_TRANSACTION_RESET!) || 0;
     this.initialResetDuration = parseInt(process.env.HAPI_CLIENT_DURATION_RESET!) || 0;
-    this.initialErrorCodes = JSON.parse(process.env.HAPI_CLIENT_ERROR_RESET || '[50]');
+    this.initialErrorCodes = JSON.parse(process.env.HAPI_CLIENT_ERROR_RESET || '[21, 50]');
```

Comment:
Q:50 is [INVALID_TRANSACTION_BODY](https://github.com/hashgraph/hedera-protobufs/blob/main/services/response_code.proto#L293) and 21 is [UNKNOWN](https://github.com/hashgraph/hedera-protobufs/blob/main/services/response_code.proto#L147)
We should document these somewhere. Likely in configs README

Reply:
Added to configuration.md